src/middlewared/middlewared/plugins/network.py

```diff
@@ -618,7 +618,7 @@ def alias_to_addr(self, alias):
         # Apply interface options specified in GUI
         if data['int_options']:
             self.logger.info('{}: applying {}'.format(name, data['int_options']))
-            proc = await Popen('/sbin/ifconfig {} {}'.format(name, data['int_options']), shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
+            proc = await Popen(['/sbin/ifconfig', name, data['int_options']], stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
```

Comment:
This does not allow setting multiple options at once. `data['int_options']` should be splitted on whitespace and concatenated to the list passed to Popen.